README.md

```diff
@@ -429,8 +426,8 @@ no matter how large. If Silk is used in production under heavy volume with large
 a huge impact on space/time performance. This behaviour can be configured with the following options:
 
 ```python
-SILKY_MAX_REQUEST_BODY_SIZE = -1  # Silk takes anything <0 as no limit
-SILKY_MAX_RESPONSE_BODY_SIZE = 1024  # If response body>1024 bytes, ignore
+SILKY_MAX_REQUEST_BYTE_SIZE = -1  # Silk takes anything <0 as no limit
+SILKY_MAX_RESPONSE_BYTE_SIZE = 1024  # If response body>1024 bytes, ignore
```

Comment:
`SILKY_MAX_REQUEST_BODY_SIZE` is being read within `django-silk`: https://github.com/search?q=repo%3Ajazzband%2Fdjango-silk%20SILKY_MAX_RESPONSE_BODY_SIZE&type=code

The variable cannot be changed to `SILKY_MAX_RESPONSE_BYTE_SIZE` in the readme without updating the code within django-silk (which would be a bad idea because it would require users of django-silk to update their code as it would be backwards incompatible)